app/controllers/syllabus_controller.rb

```diff
@@ -0,0 +1,23 @@
+class SyllabusController < ApplicationController
+
+  def download
+    course_id = params[:course_id]
+    # Get contact email for selected course
+    contact_email = Course.where(:id => course_id).limit(1).pluck(:contact_email).first
```

Comment:
I don't know as the limit here does anything - this is a select via a unique index, there are never going to be additional rows to discard.